Package.swift

```diff
@@ -8,6 +8,9 @@ let package = Package(
     ],
     products: [
         .library(name: "JWTKit", targets: ["JWTKit"]),
+        /* This target is used only for symbol mangling. It's added and removed automatically because it emits build warnings. MANGLE_START
+        .library(name: "CVaporJWTBoringSSL", type: .static, targets: ["CVaporJWTBoringSSL"]),
```

Comment:
I'd call this `CJWTKitBoringSSL` since `JWTKit` is this package's unique namespace.